ide/editor.codetemplates/src/org/netbeans/lib/editor/codetemplates/CodeTemplateInsertHandler.java

```diff
@@ -515,7 +515,7 @@ private static boolean shouldOpenCompletionAfter(TextRegionManagerEvent evt) {
             return false;
         }
         TextSync last = removed.get(removed.size()-1).activeTextSync();
-        return last.isCaretMarker() && last.isCompletionInvoke();
+        return (last != null && last.isCaretMarker() && last.isCompletionInvoke());
```

Comment:
nitpick: would you mind removing the outer braces (....)? I don't think it is helping with readability here ;)

Reply:
thanks